### ISSUE TITLE
SVGs (specifically Lottie file SVG json) not rendering properly in 13.0.3

### ISSUE THREAD

**Issue by wadehammes** [vercel/next.js#42801]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

Operating System:
      Platform: darwin
      Arch: arm64
      Version: Darwin Kernel Version 22.1.0: Sun Oct  9 20:15:09 PDT 2022; root:xnu-8792.41.9~2/RELEASE_ARM64_T6000
    Binaries:
      Node: 16.17.1
      npm: 8.15.0
      Yarn: 1.22.19
      pnpm: N/A
    Relevant packages:
      next: 13.0.3
      eslint-config-next: 13.0.3
      react: 18.2.0
      react-dom: 18.2.0

### What browser are you using? (if relevant)

Firefox

### How are you deploying your application? (if relevant)

Vercel

### Describe the Bug

After upgrading to Next 13.0.3, Lottie files (animated SVG json) are not rendering properly:

![image](https://user-images.githubusercontent.com/222170/201390146-e4e63d4f-f2e5-4161-aef9-e0aa135c7578.png)

Should look like:
![image](https://user-images.githubusercontent.com/222170/201390202-d3e980d8-8226-4c52-b720-b36e7977688a.png)

### Expected Behavior

![image](https://user-images.githubusercontent.com/222170/201390216-385a7442-cb82-4763-9f71-c4cdfee29e01.png)

Lottie files should rendering SVG properly. Not sure exactly where this regressed in the latest.

### Link to reproduction - Issues with a link to complete (but minimal) reproduction code will be addressed faster

https://rhythm-marketing-git-package-updates-111122-gotrhythm.vercel.app/

### To Reproduce

Have a site running 13.0.3 that uses Lottie files using `react-lottie-player`

**Comment by wadehammes** [comment]
Confirmed this is only happening once deployed to Vercel. SVGs render properly locally.

**Comment by jajaperson** [comment]
I'm getting a plain 404 when trying to load an svg on vercel, but no problems locally.

**Comment by sadam21x** [comment]
I ran into this bug even when running the build version locally. But there is no problem when using `next dev`

**Comment by shuvamk** [comment]
Any solution for this so far?

**Comment by balazsorban44** [comment]
@wadehammes please provide a reproduction. The URL is password protected also, but a _minimal_ code repository with easy access to the source code is always preferred to a deployed reproduction. :pray:

**Comment by wadehammes** [comment]
@balazsorban44 I don't think I will have time to create a reproduction but it seems others are having same issue, #42878 seems to have pinpointed a regression with the SWC minifier.

**Comment by shuvamk** [comment]
@balazsorban44 https://www.regexfor.com/learn on this link you can see the SVGs (lottie animations in JSON format) are not rendering properly meanwhile the local file on this repo https://github.com/shuvamk/regex-for-anything are working fine.

**Comment by balazsorban44** [comment]
For future reference, I recommend always opening an issue when a reproduction can be provided @wadehammes. We cannot investigate otherwise.

If your projects are private, please create a minimal public reproduction (preferred), or you can reach out to me by e-mail/Twitter or invite me to the repo, so we have something to go off of.

Using @shuvamk's repo, I was able to reproduce the issue :green_heart: now, so we will look into this.

**Comment by wadehammes** [comment]
@balazsorban44 definitely understand but also I don't really have the time in my day to create reproductions of things outside my control, too much on my plate with work. There were enough other people here with same issue so I was hoping someone would come through, appreciate everything you guys are doing.

**Comment by cangel111** [comment]
I ran into the same issue.

In Next.js v13  the SWC Minifier is enabled by default. https://nextjs.org/docs/advanced-features/compiler

You can fix it, at least temporarily, by adding `swcMinify: false` to your nextConfig.

```
// next.config.js

module.exports = {
  swcMinify: false,
}
```

**Comment by jordanlambrecht** [comment]
Also experiencing this issue. I can also confirm that the issue only persists in production. A great example is directly on the landing page for the two links below. The background SVGs are powered by Lottie like OP mentioned, the foreground graphic is a straight-up SVG.

Next.js build with 13.0.3: https://pixelbakery-website-c6qliha18-pixelbakery.vercel.app/
Next.js build with 13.0.2: https://pixelbakery.com
source files: https://github.com/pixelbakery/pixelbakery-website

**Comment by DavidChouinard** [comment]
We're also having this issue

**Comment by balazsorban44** [comment]
Anyone having the same issue, please +1: the first comment unless you are adding a minimal reproduction (although we already confirmed the problem and are working on it). Thank you!

**Comment by smitpatelx** [comment]
Fix is merge, but not yet released as of `V 13.0.4`, so for now the only option is to disable swc_minify. I wish there was an exclusion option in nextjs, `*/.json` to disable minifying json files or a single file.
